src/Field/BasicField.php

```diff
@@ -115,6 +115,12 @@ public function fetchFieldKey($fieldName)
      */
     public function fetchFieldType($fieldKey)
     {
+        // Check if this a clone field. If it is, return the field type of its clone.
+        $fieldKeyIds = explode('_', $fieldKey);
+        if (count($fieldKeyIds) == 4) {
```

Comment:
Hi @njbarrett how are you considering 4 tokens here? If I have a field called `first_name`? Does it still work?